packages/ponytest/pony_test.pony

```diff
@@ -242,19 +242,20 @@ actor PonyTest
     """
     Run the given test, subject to our filters and options.
     """
+    
```

Comment:
can you kill this? we generally start the first line of the method abutting the doc string.